test/example.spec.ts

```diff
@@ -1,26 +1,27 @@
+///<reference path="../node_modules/@types/mocha/index.d.ts"/>
```

Comment:
This way of referencing types has been deprecated since TypeScript 2. It is now smart enough to use the `@types/mocha` package.